velox/expression/fuzzer/ExpressionFuzzerVerifier.cpp

```diff
@@ -308,6 +308,9 @@ void ExpressionFuzzerVerifier::go() {
   size_t i = 0;
   size_t numFailed = 0;
 
+  // TODO: some expression will throw exception for NaN input, eg: IN predicate
+  // for floating point. remove this constraint once that are fixed
+  vectorFuzzer_->setConstraint(Constraint{true, true});
```

Comment:
Move this to getVectorFuzzerOptions() in FuzzerRunner.cpp and add a new flag --enable_nan_and_inf to control it more conveniently.

Reply:
This can not be done because the constraint is per function based instead of a global setting

Reply:
This line 313 is a global setting in expression fuzzer. You can move VectorFuzzer::constraint_ into VectorFuzzer::opts_::floatingPointConstraint_, then change getVectorFuzzerOptions() that is only used in expression fuzzer as follows:
```
VectorFuzzer::Options getVectorFuzzerOptions() {
  VectorFuzzer::Options opts;
  opts.vectorSize = FLAGS_batch_size;
  opts.stringVariableLength = true;
  opts.stringLength = 100;
  opts.nullRatio = FLAGS_null_ratio;
  opts.floatingPointConstraint_ = FLAGS_enable_nan_and_inf ? Constraint(false, false) : Constraint(true, true);
  return opts;
}
```

By the way, could we make the Constraint struct to use flags to indicate inclusion instead of exclusion? The logic is more straightforward this way. E.g.,
```
struct Constraint {
  bool includeNaN;
  bool includeInfinity;
};
```